src/database/sql/sequelize/mappers/clinical/careplan/enrollment.mapper.ts

```diff
@@ -23,6 +23,7 @@ export class EnrollmentMapper {
             PlanName            : enrollment.PlanName,
             StartAt             : enrollment.StartDate,
             EndAt               : enrollment.EndDate,
+            StopAt              : enrollment.StopAt,
```

Comment:
Please change from 'StopAt' to 'StoppedAt' as this column will be updated only after enrollment is stopped.

Reply:
Changed the column name.